includes/Blocks/Block.php

```diff
@@ -61,10 +61,23 @@ public function __construct( WP_Block_Type $block, Registry $block_registry ) {
 		$this->block_registry   = $block_registry;
 		$this->block_attributes = $this->block->attributes;
 		$this->type_name        = $this->format_type_name( $block->name );
-
+		$this->update_scalars();
 		$this->register_block_type();
 	}
 
+	/**
+	 * Update scaler types
+	 * 
+	 * @return void
+	 */
+	public function update_scalars() {
+		Registry::register_graphql_scalar( 'BlockAttributesObject', [ 
```

Comment:
There is no `Registry::register_graphql_scalar` call and the test cases are failing. According to the [documentation](https://www.wpgraphql.com/functions/register_graphql_scalar) it should be `register_graphql_scalar`